python/datashaper/datashaper/engine/verbs/convert.py

```diff
@@ -85,18 +86,32 @@ def _to_datetime(column: pd.Series) -> pd.Series:
     return pd.to_datetime(column, errors="coerce")
 
 
+def _to_array(column: pd.Series, delimiter: str) -> pd.Series | pd.DataFrame:
+    def convert_value(value: Any) -> list:
+        if pd.isna(value):
+            return []
+        if isinstance(value, list):
+            return value
+        if isinstance(value, str):
+            return value.split(delimiter)
+        return []

```

Comment:
Wonder if the default should instead be [value]

Reply:
Yeah good call